tests/Jellyfin.Networking.Tests/NetworkTesting/Jellyfin.Networking.Tests.csproj

```diff
@@ -13,16 +13,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.0-preview-20201020-06" />
```

Comment:
Unrelated changes?

Reply:
Did it on purpose - VS wasn't detecting all the tests. Found a resolution online to say upgrade testing packages. Did and it worked - so left it.

Can try to roll back?

Reply:
Try just running `dotnet test` if VS doesn't detect all of the tests